src/librustdoc/html/static/js/search.js

```diff
@@ -227,6 +634,12 @@ window.initSearch = function(rawSearchIndex) {
         }
 
         function sortResults(results, isType) {
+            var nameSplit = null;
+            if (parsedQuery.elems.length === 1) {
+                var hasPath = typeof parsedQuery.elems[0].path === "undefined";
+                nameSplit = hasPath ? null : parsedQuery.elems[0].path;
+            }
```

Comment:
`nameSplit` is defined here, but first used 100 lines below. Can you move this lower so it is defined just outside the `for` loop where it is used?

Reply:
Good idea!